pygmt/src/text.py

```diff
@@ -147,6 +147,8 @@ def text_(
     {f}
     {p}
     {t}
+        *transparency* can also be a 1d array to set varying transparency
+        for texts, but this option is only valid if using x/y.
```

Comment:
Similarly, should be `x/y/text` here.
```suggestion
        for texts, but this option is only valid if using x/y/text.
```